docs/easy-deployments.md

```diff
@@ -0,0 +1,38 @@
+# Easy Deployments
+
+There are several easy deployments instructions described in this document to help you deploy your fork of the ott-web-app to some of the popular web application hosting platforms.
```

Comment:
If we are suggesting that people fork this repo, we might want to include  a link to how to fork a repo here (or in the build from source docs). Here's github's link: https://docs.github.com/en/get-started/quickstart/fork-a-repo